ee/modules/650-runtime-audit-engine/crds/falco-audit-rules.yaml

```diff
@@ -0,0 +1,139 @@
+              requiredK8sAuditPluginVersion:
+                type: string
+                description: |
+                  Used to track compatibility between rules content and plugin versions.
+                pattern: '^[1-9]\.+[0-9\.]*[0-9]$'
+              rules:
+                type: array
+                minLength: 1
+                description: |
+                  Describes Falco rules, that will be applied to monitor the runtime of the cluster.
+                  These rules helps detecting threats at runtime by observing the behavior of your applications and containers.
+
+                  Follow [the Falco documentation](https://falco.org/docs/rules/) and [reference](https://falco.org/docs/reference/rules/) for more details.
+                items:
+                  type: object
+                  oneOf:
+                  - required: ["macro"]
+                  - required: ["list"]
+                  - required: ["rule"]
+                  properties:
+                    rule:
+                      type: object
+                      description: |
+                        Conditions under which an alert should be generated.
+                        A rule is accompanied by a descriptive output string that is sent with the alert.
+                      required:
+                      - "name"
+                      - "condition"
+                      - "desc"
+                      - "output"
+                      - "priority"
+                      properties:
+                        name:
+                          type: string
+                          description: |
+                            A short, unique name for the rule.
+                        condition:
+                          type: string
+                          description: |
+                            A filtering expression that is applied against events to check whether they match the rule.
+                        desc:
+                          type: string
+                          description: |
+                            A longer description of what the rule detects.
+                        output:
+                          type: string
+                          description: |
+                            Specifies the message that should be output if a matching event occurs.
+                        priority:
+                          type: string
+                          description: |
+                            A case-insensitive representation of the severity of the event.
+                          enum: ["Emergency", "Alert", "Critical", "Error", "Warning", "Notice", "Informational", "Debug"]
+                        enabled:
+                          type: boolean
+                          default: true
+                          description: |
+                            If set to `false`, a rule is neither loaded nor matched against any events.
```

Comment:
As far as I know, it helps debug rules or disable flaky ones in runtime. I chose this option because 
1. The original Falco rules design has it.
2. In Kubernetes, you cannot comment on the part of a resource without saving it to a TMP file. It seems like a good feature to disable a rule temporarily right in the file without deleting it from the spec.